### ISSUE TITLE
WallPanel is not automatically loaded after phone restart (v0.9.3-beta.4 & v0.9.3-beta.6)

### ISSUE THREAD

**Issue by BoneheadFraggle** [thanksmister/wallpanel-android#247]
**Describe the bug**
Even though Open On Device Boot = On, the app is not loaded after rebooting the device.

**To Reproduce**
1. Configure WallPanel to use Open On Device Boot
2. Restart the phone
3. Wait for it...
4. WallPanel is never started

**Expected behavior**
Open On Device Boot = On, should start the app when "ON BOOT COMPLETED"

**Screenshots**
N/A

**Smartphone (please complete the following information):**
 - Device: OnePlus One, Nexus 6p, Sony Z3
 - OS: Android 8, 10, 11
 - Browser Stock browser

**Additional context**
I installed Auto Start and StaY! which starts WallPanel instead, but it would be really nice if it worked as before.

**Comment by thanksmister** [comment]
This works on my older devices, but it seems Google has made this a real complicated mess on older devices. Also some devices use their own boot manager so they need some special settings for this on the device.   I suspect on newer devices, this is blocked by the device settings.  More on this here https://stackoverflow.com/questions/20441308/boot-completed-not-working-android.

**Comment by BoneheadFraggle** [comment]
I tried to be really accurate when I tried to iron out which configurations that worked yesterday, but when I try the same stuff today, I get other results. Must be my devices or network or Home Assistant or..
However, in while trying out everything again, I found another thing, but that's another issue.
Closing this.
